### ISSUE TITLE
Is HospitalRun development active?

### ISSUE THREAD

**Issue by prd-infinity** [HospitalRun/hospitalrun#475]
## 💬 Questions and Help
Is the development of HospitalRun currently active? I don't any recent code comment. Also many of the tools are not working (for example Slack, demo app,etc., )

**Comment by matteovivona** [comment]
No, unfortunately not. It's too big as project and needs multiple full-time senior developers
